install_openssl.sh

```
@@ -0,0 +1,37 @@
+wget https://www.openssl.org/source/openssl-1.1.1g.tar.gz.sha256
+
+sha256sum openssl-1.1.1g.tar.gz
+
+cat openssl-1.1.1g.tar.gz.sha256
+
+tar zxvf openssl-1.1.1g.tar.gz
+
+# tfw python3.9 shell
+webservice python3.9 shell
+
+cd openssl-1.1.1g
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)